package.json

```diff
@@ -238,6 +238,7 @@
     "@babel/preset-react": "7.22.15",
     "@babel/preset-typescript": "7.23.0",
     "@btckit/types": "0.0.19",
+    "@leather-wallet/prettier-config": "^0.0.1",
```

Comment:
needs to be exact package (can use -E when installing)

Reply:
thanks! I was a bit too hasty requesting review here 😊